### PR TITLE
Added tip on Intel Optane persistent memory (bsc#1201838)

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Description

Intel Optane has specific modes for its persistent memory model. This update reflects this.

### PR creator: Are there any relevant issues/feature requests?

* bsc#1201838

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [x] SLE 15 SP1
  - [ ] SLE 15 SP0
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by ajaeger** [comment]
LGTM, thanks!

File: xml/hardware_nvdimm.xml [hunk 1]
```diff
@@ -105,7 +105,54 @@
...
+     <para>
+      App Direct Mode has been supported since SLE 12 SP4.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      In <emphasis>Memory Mode</emphasis>, the Intel Optane memory serves as a
+      cost-effective, high-capacity alternative to DRAM. In this mode, separate
+      DRAM DIMMs act as a cache for the most frequently-accessed data while the
+      Optane DIMMs memory provide large memory capacity. However, compared with
```
Comment (dariavladykina) [inline xml/hardware_nvdimm.xml:130]:
```suggestion
      DRAM DIMMs act as a cache for the most frequently accessed data while the
      Optane DIMMs memory provides large memory capacity. However, compared with
```

File: xml/hardware_nvdimm.xml [hunk 2]
```diff
@@ -105,7 +105,54 @@
...
+  <tip>
+   <title>Intel Optane DC Persistent Memory</title>
+   <para>
+    Intel Optane DIMMs memory can be used in specific modes:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      In <emphasis>App Direct Mode</emphasis>, the Intel Optane memory is used as
+      a fast persistent storage, an alternative to SSDs and NVMe devices. Data in
```
Comment (dariavladykina) [inline xml/hardware_nvdimm.xml:118]:
```suggestion
      fast persistent storage, an alternative to SSDs and NVMe devices. Data in
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please consider some minor suggestions.

**Pull request merged**